### PR TITLE
[HDFS] Add configurable administrators

### PR DESCRIPTION
**Pull request by gabrielhartmann**

### CODE REVIEW

File: frameworks/hdfs/universe/config.json [hunk 1]
```diff
@@ -204,6 +204,11 @@
       "type": "object",
       "description": "HDFS File System configuration options",
       "properties": {
+        "administrators": {
+          "type": "string",
+          "description": "Administrators for the HDFS cluster",
+          "default": "core,centos"
```
Comment (loren) [inline frameworks/hdfs/universe/config.json]: consider adding `azureuser`, too.

**Comment by gabrielhartmann** [comment]
@loren:  Is this Elastic test failure due to slowness i.e. a flake or is this a real bug?  The PR doesn't touch Elastic.

**Comment by loren** [comment]
It's a flake (that particular plugin install/uninstall test has been passing for months now). Looking at the logs, it looks like the call to `marathon_update` doesn't happen successfully for one reason or another. I'd expect to see evidence of this in the logs:

```
requests.put(marathon_api_url('apps/{}'.format(PACKAGE_NAME)), json=config, headers=REQUEST_HEADERS, verify=False)
```

Since the marathon config update doesn't go through, the plugin never gets installed, so the test spins and times out.

**Comment by gabrielhartmann** [comment]
retest this please

**Pull request merged**